src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1784,29 +1861,29 @@
 helps['network vnet peering'] = """
     type: group
     short-summary: Manage peering connections between Azure Virtual Networks.
+    long-summary: To learn more about VNet Peering visit https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-manage-peering
```

Comment:
Unless `VNet Peering` is a product, write as `virtual network peering`.

Reply:
Fixed in my branch